source/includes/concept/_overview.md

```diff
@@ -222,4 +222,104 @@ The following endpoints adhere to the Filtering conventions defined above:
 * [Get all widgets endpoint](/index-rw.html#getting-all-widgets)
 * [Get all users endpoint](/index-rw.html#getting-all-users)
 
+### Pagination
+
+> Example request where default pagination is applied, returning one page of 10 elements (1st - 10th elements):
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset
+```
+
+> Example request fetching the 3rd page of 10 elements (21st - 30th elements):
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?page[number]=3
+```
+
+> Example request fetching the 5th page of 20 elements (81st - 100th elements):
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?page[number]=5&page[size]=20
+```
+
+Many times, when you're calling RW API's list endpoints, there will be a lot of results to return. Without pagination, a simple search could return hundreds or even thousands of elements, causing extraneous network traffic. For that reason, many services list their resources as pages, to make sure that not only responses are easier to handle, but also that services are scalable. Paged results have a built-in default limit of 10 elements, but we recommend you always explicitly set the limit parameter to ensure you know how many results per page you'll get. 
```

Comment:
Paged results have a built-in default limit of 10 elements -> Most paged results have a built-in default limit of 10 elements

Also, not sure if "paginated" would be better, as I think that's the term we use in other places on the doc, but up to you